_includes/contents.html

```diff
@@ -0,0 +1,129 @@
+<div class="table-of-contents-inner">
+
+{% for item in site.data.pages %}
+
+    {% if item.categories %}   
+
+        {% if page.url == item.url %}
+            
+            {% if item.subpageitems %}
+                <h6>Contents</h6>
```

Comment:
There's still plenty of duplication here in this file.

Reply:
> If there are no tags perhaps the tag ui shouldn't be shown (this is from /documentation/develop/)
> 
> <img alt="Develop___Extension_Workshop" width="747" src="https://user-images.githubusercontent.com/1514/60972349-ffa6a080-a31d-11e9-9341-5a61145722c8.png">

If the tag isn't defined in the front matter, it won't show. I left an empty array in the front matter on those pages to highlight to the content team that they will need to fill in those tags (I believe that they are in the process of coming up with tag content for those pages)

Reply:
Ah ok, fair enough 👍